src/olympia/addons/templates/addons/persona_preview.html

```diff
@@ -25,7 +25,7 @@
             {# L10n: {0} is the number of users. #}
             {{ ngettext("<strong>{0}</strong> user",
                         "<strong>{0}</strong> users",
-                        addon.persona.popularity)|f(addon.persona.popularity|numberfmt)|safe }}
+                        addon.persona.popularity)|fe(addon.persona.popularity|numberfmt)|safe }}
```

Comment:
In this specific case `|safe` should mark `addon.persona.popularity` as safe but we won't need that you're right.

Test + documentation sounds good, I'll add that.